ClientSide(FrontEnd)/clay_/src/components/Images.css

```diff
@@ -0,0 +1,263 @@
+    padding: 20px;
+    margin: 30px;
+}
+.img6{
+    width:450px;
+    border: 2px solid white;
+    height: 400px;
+}
+.nextpage1{
+    width: 400px;
+     height:400px;
+    border: 2px solid gray;
+    margin-left: 750px;
+    margin-top: -450px;
+    text-align: center;
+    color: black;
+}
+.nextpage1 > p{
+    margin-top: 40px;
+    color: black;
+}
+.nextpage1 > button{
+    padding: 20px;
+    border-radius: 35px;
+    margin-top: 40px;
+    background-color: antiquewhite;
+    border: 1px solid antiquewhite;
+}
+.icons{
+    display: flex;
+    justify-content: space-evenly;
+    margin-top: 150px;
+    border: 2px solid gray;
+    height: 500px;
+    background-color: gray;
+}
+.ptag{
+    color: black;
+    text-align: center;
+}
+.img7{
+    width: 250px;
+    height: 250px;
+    border-radius: 130px;
+    margin-top: 80px;
+}
+.img8{
+    width: 250px;
+    height: 250px;
+    border-radius: 130px;
+    margin-top: 80px;
+
+}
+.img9{
+    width: 250px;
+    height: 250px;
+    border-radius: 130px;
+    margin-top: 80px;
+
+}
+.img10{
+    width: 250px;
+    height: 250px;
+    border-radius: 130px;
+    margin-top: 80px;
+
+}
+.maindiv{
+    height: 400px;
+    border: 2px solid gray;
+    margin: 80px 200px 10px 200px;
+    display: flex;
+
+}
+.firstdiv{
+    height: 400px;
+    border: 2px solid gray;
+    width: 550px;
+    background-color: gray;
+   
+}
+.seconddiv{
+    height: 400px;
+    border: 2px solid gray;
+    width: 490px;
+}
+.img11{
+    width: 200px;
+    height: 300px;
+    
+}
+.img12{
+    width: 200px;
+    height: 300px;
+    margin-left: 280px;
+}
+.img13{
+    width: 200px;
+    height: 300px;
+    margin-left: 450px;
+    margin-top: 130px;
+}
+.img14{
+    width: 200px;
+    height: 200px;
+    margin-bottom: 1000px;
+    margin-left: 150px;
+}
+.middlebox{
+    margin: 10px 500px 10px 500px;
+    height: 400px;
+    margin-top: -410px;
+}
+.alltext{
+    text-align: center;
+    margin-top: -1010px;
+    width: 400px;
+    height: 400px;
+    margin-left: 60px;
+}
+.alltext > button{
+    padding: 10px;
+}
+.footer{
+    background-color: rgb(18, 9, 9);
+    width: 70%;
+    height: 40%;
+    margin-left: 200px;
+}
+.footer1{
+        background-color: gray;
+        height: 40%;
+        display: flex;
+        padding: 70px;
+        margin-top: 400px;
+}
+.service{
+    margin-left: 150px;
+    font-size: 20px;
+}
+.contact{
+    margin-left: 150px;
+    font-size: 20px;
+}
+.social{
+    margin-left: 150px;
+    font-size: 20px;
+}
+.about{
+    margin-left: 50px;
+    font-size: 20px; 
+}
+.logo > img{
+    width: 60px;
+    height: 60px;
+    margin-left: -180px;
+    margin-top: 300px;
+    /* margin: -120px; */
+    border-radius: 40px;
+}
+.subscribe{
+    width: 320px;
+    height: 60px;
+    border: 2px solid red;
+    border-radius: 40px;
+    margin-left: 60px;
+}
+.img19{
+    width: 200px;
+    height: 400px;
+    margin-left: 260px;
+    margin-top: -80px;
+}
```

Comment:
add one empty line at the end of your code

Reply:
i added one empty line